cmd/hyperkube/hyperkube.go

```diff
@@ -162,7 +162,7 @@ func (hk *HyperKube) Run(args []string) error {
 		hk.Usage()
 		return err
 	}
-
+	s.Flags().SetInterspersed(false)
```

Comment:
This is a behavior change.  Some of the servers may want to allow for interspersed flags.  No reason to slam the preference here.